src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
```

Comment:
```suggestion
  const cost = days * 40;
```
Cost of what? Let's make it clear